x/xibc/core/packet/keeper/packet.go

```diff
@@ -122,42 +123,10 @@ func (k Keeper) RecvPacket(ctx sdk.Context, msg *types.MsgRecvPacket) error {
 
 	_ = ctx.EventManager().EmitTypedEvent(
 		&types.EventRecvPacket{
-			Sequence:   fmt.Sprintf("%d", packet.GetSequence()),
-			SrcChain:   packet.GetSourceChain(),
-			DstChain:   packet.GetDestChain(),
-			RelayChain: packet.GetRelayChain(),
-			Ports:      packet.GetPorts(),
-			DataList:   packet.GetDataList(),
+			Packet: msg.Packet,
 		},
 	)
 
-	if packet.GetRelayChain() == chainName {
```

Comment:
if (dest_chain != chain_name && dest_client_exist) {
    handle relay chain logic
}

Reply:
If we have the work for relay packet by teleport to be the relayChain . relayer may only relay the packet with the channel(srcChain -> destChain) , how do relayer know that the teleport is relayChain and the teleport not the srcChain and destChain?